pages/login.tsx

```diff
@@ -1,16 +1,83 @@
-import React from 'react';
-import LoginForm from '../components/LoginForm/LoginForm';
-
-function Login() {
-    return (
-        <div className="page-margin-top">
-            <div className="container">
-                <div className="row justify-content-center align-items-center">
-                    <LoginForm />
-                </div>
-            </div>
+import { useState, useEffect, useContext } from 'react'
+import Router from 'next/router'
+import { magic } from '../lib/Magic/magic'
+import LinkedInLogin from '../components/LinkedInLogin'
+import { useForm } from 'react-hook-form'
+import { UserContext } from '../lib/userContext'
+import EmailLogin from '../components/EmailLogin';
+import { authenticateWithServer } from '../lib/Magic/magicFunctions'
+
+
+const Login = () => {
+  const [disabled, setDisabled] = useState(false)
+  const [user,setUser] = useContext(UserContext);
+  // const [user, setUser] = useState(null)
+
+  const onSubmit = async (e: object) => {
+    // e.preventDefault();
+    // await primsa db response
+    console.log(e)
+  }
+
+  const validate = (values: object) => {
+    return {}
+  }
+
+  const { register, handleSubmit } = useForm({
+    // onSubmit,
+    // validate
```

Comment:
There's various things commented out in this file - can they be removed or is there a reason to keep them? Same question for the `console.log()` statements.

Reply:
Removed old code related to react-hook-forms , thats no longer being used.